<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StringConnection</class>
 <widget class="QMainWindow" name="StringConnection">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>730</width>
    <height>200</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>730</width>
    <height>200</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>730</width>
    <height>200</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QFrame" name="fram_content">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QFrame" name="frame_3">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <widget class="QPushButton" name="btn_testConnection">
          <property name="geometry">
           <rect>
            <x>461</x>
            <y>88</y>
            <width>150</width>
            <height>45</height>
           </rect>
          </property>
          <property name="minimumSize">
           <size>
            <width>150</width>
            <height>45</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>150</width>
            <height>45</height>
           </size>
          </property>
          <property name="baseSize">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>MS Shell Dlg 2</family>
            <pointsize>11</pointsize>
            <italic>false</italic>
            <bold>false</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton{
	border-style: none;
	border-width: 2px;
	font: 75 11pt &quot;MS Shell Dlg 2&quot;;
	color: rgb(255, 255, 255);
	background-color: #4195FF;
	border-radius: 12px;
	border-bottom-color: rgb(4, 35, 38);
	border-right-color:  rgb(4, 35, 38);
}
QPushButton:hover{
	border-style: none;
	border-width: 2px;
	font: 75 11pt &quot;MS Shell Dlg 2&quot;;
	color: rgb(255, 255, 255);
	background-color: #0954B2;
	border-radius: 12px;
	border-bottom-color: rgb(4, 35, 38);
	border-right-color:  rgb(4, 35, 38);
}


QPushButton:pressed{
	border-style: insetset;
	border-width: 2px;
	font: 75 11pt &quot;MS Shell Dlg 2&quot;;
	color: rgb(255, 255, 255);
	background-color: #4195FF;
	border-radius: 8px;
	border-top-color: rgb(4, 35, 38);
	border-left-color:  rgb(4, 35, 38);
}</string>
          </property>
          <property name="inputMethodHints">
           <set>Qt::ImhNone</set>
          </property>
          <property name="text">
           <string>Testar conexão</string>
          </property>
          <property name="icon">
           <iconset>
            <normaloff>../src/icon_uncheck.svg</normaloff>
            <normalon>../src/icon_uncheck.svg</normalon>../src/icon_uncheck.svg</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>20</width>
            <height>20</height>
           </size>
          </property>
         </widget>
         <widget class="QPushButton" name="btn_exit">
          <property name="geometry">
           <rect>
            <x>620</x>
            <y>41</y>
            <width>100</width>
            <height>45</height>
           </rect>
          </property>
          <property name="minimumSize">
           <size>
            <width>100</width>
            <height>45</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>100</width>
            <height>45</height>
           </size>
          </property>
          <property name="baseSize">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>MS Shell Dlg 2</family>
            <pointsize>11</pointsize>
            <italic>false</italic>
            <bold>false</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton{
	border-style: none;
	border-width: 2px;
	font: 75 11pt &quot;MS Shell Dlg 2&quot;;
	color: rgb(255, 255, 255);
	background-color: #F2163E;
	border-radius: 12px;
	border-bottom-color: rgb(4, 35, 38);
	border-right-color:  rgb(4, 35, 38);
}
QPushButton:hover{
	border-style: none;
	border-width: 2px;
	font: 75 11pt &quot;MS Shell Dlg 2&quot;;
	color: rgb(255, 255, 255);
	background-color: #BF0404;
	border-radius: 12px;
	border-bottom-color: rgb(4, 35, 38);
	border-right-color:  rgb(4, 35, 38);
}


QPushButton:pressed{
	border-style: insetset;
	border-width: 2px;
	font: 75 11pt &quot;MS Shell Dlg 2&quot;;
	color: rgb(255, 255, 255);
	background-color: #4195FF;
	border-radius: 8px;
	border-top-color: rgb(4, 35, 38);
	border-left-color:  rgb(4, 35, 38);
}</string>
          </property>
          <property name="inputMethodHints">
           <set>Qt::ImhNone</set>
          </property>
          <property name="text">
           <string>Sair</string>
          </property>
          <property name="icon">
           <iconset>
            <normaloff>../../Utilitario/src/icon_cancel.svg</normaloff>../../Utilitario/src/icon_cancel.svg</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>20</width>
            <height>20</height>
           </size>
          </property>
         </widget>
         <widget class="QLabel" name="lb_localDb">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>39</y>
            <width>201</width>
            <height>31</height>
           </rect>
          </property>
          <property name="text">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:16pt; font-weight:700;&quot;&gt;Local de Instalação:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
         <widget class="QLineEdit" name="le_localDb">
          <property name="geometry">
           <rect>
            <x>210</x>
            <y>41</y>
            <width>241</width>
            <height>35</height>
           </rect>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>35</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>35</height>
           </size>
          </property>
          <property name="sizeIncrement">
           <size>
            <width>200</width>
            <height>35</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">font: 16pt &quot;Segoe UI&quot;;</string>
          </property>
          <property name="placeholderText">
           <string>Name Database</string>
          </property>
         </widget>
         <widget class="QPushButton" name="btn_instalation">
          <property name="geometry">
           <rect>
            <x>460</x>
            <y>41</y>
            <width>150</width>
            <height>45</height>
           </rect>
          </property>
          <property name="minimumSize">
           <size>
            <width>150</width>
            <height>45</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>150</width>
            <height>45</height>
           </size>
          </property>
          <property name="baseSize">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>MS Shell Dlg 2</family>
            <pointsize>11</pointsize>
            <italic>false</italic>
            <bold>false</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton{
	border-style: none;
	border-width: 2px;
	font: 75 11pt &quot;MS Shell Dlg 2&quot;;
	color: rgb(255, 255, 255);
	background-color: #4195FF;
	border-radius: 12px;
	border-bottom-color: rgb(4, 35, 38);
	border-right-color:  rgb(4, 35, 38);
}
QPushButton:hover{
	border-style: none;
	border-width: 2px;
	font: 75 11pt &quot;MS Shell Dlg 2&quot;;
	color: rgb(255, 255, 255);
	background-color: #0954B2;
	border-radius: 12px;
	border-bottom-color: rgb(4, 35, 38);
	border-right-color:  rgb(4, 35, 38);
}


QPushButton:pressed{
	border-style: insetset;
	border-width: 2px;
	font: 75 11pt &quot;MS Shell Dlg 2&quot;;
	color: rgb(255, 255, 255);
	background-color: #4195FF;
	border-radius: 8px;
	border-top-color: rgb(4, 35, 38);
	border-left-color:  rgb(4, 35, 38);
}</string>
          </property>
          <property name="inputMethodHints">
           <set>Qt::ImhNone</set>
          </property>
          <property name="text">
           <string>Local de instalação</string>
          </property>
          <property name="icon">
           <iconset>
            <normaloff>../src/icon_folder.svg</normaloff>../src/icon_folder.svg</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>20</width>
            <height>20</height>
           </size>
          </property>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="frame_2">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>55</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>55</height>
       </size>
      </property>
      <property name="sizeIncrement">
       <size>
        <width>0</width>
        <height>10</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: #0954B2;
border-radius: 2px;
</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="spacing">
        <number>30</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>10</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="lb_commom">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>35</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>35</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">font: 75 9pt &quot;MS Shell Dlg 2&quot;;
color: #fff;
</string>
         </property>
         <property name="text">
          <string>Utilitario de serviços - versão 0.0.1 | Campo Grande -MS | 20:19</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btn_statusConection">
         <property name="minimumSize">
          <size>
           <width>45</width>
           <height>45</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>45</width>
           <height>45</height>
          </size>
         </property>
         <property name="baseSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>MS Shell Dlg 2</family>
           <pointsize>11</pointsize>
           <italic>false</italic>
           <bold>false</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton{
	border-style: none;
	border-width: 2px;
	font: 75 11pt &quot;MS Shell Dlg 2&quot;;
	color: rgb(255, 255, 255);
	background-color: #0954B2;
	border-radius: 12px;
	border-bottom-color: rgb(4, 35, 38);
	border-right-color:  rgb(4, 35, 38);
}
QPushButton:hover{
	border-style: none;
	border-width: 2px;
	font: 75 11pt &quot;MS Shell Dlg 2&quot;;
	color: rgb(255, 255, 255);
	background-color: #0954B2;
	border-radius: 12px;
	border-bottom-color: rgb(4, 35, 38);
	border-right-color:  rgb(4, 35, 38);
}


QPushButton:pressed{
	border-style: insetset;
	border-width: 2px;
	font: 75 11pt &quot;MS Shell Dlg 2&quot;;
	color: rgb(255, 255, 255);
	background-color: #4195FF;
	border-radius: 8px;
	border-top-color: rgb(4, 35, 38);
	border-left-color:  rgb(4, 35, 38);
}</string>
         </property>
         <property name="inputMethodHints">
          <set>Qt::ImhNone</set>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>../src/icon_disconnectDB.svg</normaloff>../src/icon_disconnectDB.svg</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>50</width>
           <height>50</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
